airbyte-webapp/src/components/ui/Toast/Toast.module.scss

```diff
@@ -1,3 +1,89 @@
+@use "scss/colors";
+@use "scss/variables" as vars;
+
+$toast-icon-size: 13px;
+$toast-icon-container-size: 34px;
+
+@keyframes slide-up-animations {
+  0% {
+    transform: translate(-50%, -100%);
+    bottom: -49px;
+  }
+
+  100% {
+    transform: translate(-50%, 0);
+    bottom: 49px;
+  }
+}
+
+@mixin type($name, $color, $background) {
+  &.#{$name} {
+    background-color: $background;
+    border: 1px solid $color;
+
+    .iconContainer {
+      background-color: $color;
+    }
+
+    .toastIcon {
+      color: $color;
+    }
+  }
+}
+
+.toastContainer {
```

Comment:
I noticed one more thing: It needs the shadow mixin

Reply:
Oh, yeah, missed that 👍 fixed